### PR TITLE
Merge 2.2

### PR DESCRIPTION
**Pull request by soyuka**

### CODE REVIEW

**Comment by teohhanhui** [comment]
Let's wait for #1993

**Comment by soyuka** [comment]
let me add it :)

File: tests/Annotation/ApiResourceTest.php [hunk 1]
```diff
@@ -102,12 +102,11 @@ public function testApiResourceAnnotation()
         ], $resource->attributes);
     }
 
-    /**
-     * @expectedException \ApiPlatform\Core\Exception\InvalidArgumentException
-     * @expectedExceptionMessage Unknown property "invalidAttribute" on annotation "ApiPlatform\Core\Annotation\ApiResource".
-     */
     public function testConstructWithInvalidAttribute()
     {
+        $this->expectException(\ApiPlatform\Core\Exception\InvalidArgumentException::class);
```
Comment (teohhanhui) [inline tests/Annotation/ApiResourceTest.php]: Should add `import` for this.

**Comment by bendavies** [comment]
#1993 is in!

**Comment by soyuka** [comment]
let's wait for #2004 :D

**Comment by teohhanhui** [comment]
#2004 is merged. :smile:

**Comment by teohhanhui** [comment]
Resolving conflicts should be done in the merge commit or further commits, without rewriting history. :stuck_out_tongue:

**Pull request merged**